### DISCUSSION TITLE
Emails sent from Document Repository - Are these logged anywhere?

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1365] [Q&A / Help] [answered]
Hi, thanks for your time.

Just wondering also, if there is a way to view the emails that have been sent from the Document Repository. Like a log or report? Any suggestions.

Cheers!

**Comment by bobintetley** [comment, marked as answer]
They're recorded in the audit trail. You can view them with the report "Audit Trail: Sent Emails by Date"
